client/components/chart/utils.js

```diff
@@ -493,10 +502,16 @@ export const drawLines = ( node, data, params ) => {
 			} )
 			.attr( 'cx', d => params.xLineScale( new Date( d.date ) ) )
 			.attr( 'cy', d => params.yScale( d.value ) )
-			.on( 'mouseover', ( d, i, nodes ) =>
-				handleMouseOverLineChart( d, i, nodes, node, data, params )
+			.attr( 'tabindex', '0' )
+			.attr(
+				'aria-label',
+				d => `${ getTooltipDate( params, d.date ) } ${ d.key } ${ formatCurrency( d.value ) }`
```

Comment:
Having `key` in this label makes the label too long & a bit confusing on reports, for example I hear: `August 06 2018 Last Month (Aug 1 - 31, 2018) $8,080.81` – it's unclear what that dollar value represents, the entire month or just that day? Since each line is in a region with the key label, I think we can omit it here.